### ISSUE TITLE
LibreOffice-printer not detected

### ISSUE THREAD

**Issue by mapommfj** [flathub/org.libreoffice.LibreOffice#307]
OS=LinuxMint22, Kernel 6.8.0-47, drivers Canon for MG3650S, Flatpak 24.8.2.1 (x86_64), Desktop Cinnamon

my printer is not detected, always 'Generic Printer' which don't work.

AppImage (24.8.2.1) and non-flatpak version of Libreoffice (4:24.2.6-ubuntu0.24.04.1) works well.

**Comment by zebop917** [comment]
Just noting that I had the same issue.
Linux Mint 22 Cinnamon, kernel 6.8.0-48, Canon MG4200 series, driverless, 2.0.0
Flatpak 1.14.6 - this just shows 'Generic Printer'
With the current System Package (4:24.2.6-0ubuntu0.24.04.1), the printer is correctly identified.

**Comment by Erick555** [comment]
does it work when you start it with `flatpak run --socket=cups org.libreoffice.LibreOffice`?

**Comment by mapommfj** [comment]
Yes it works with this command. I have not printed but my printer is in the list of LibreOffice.

**Comment by Erick555** [comment]
You can make it permanent with `flatpak override --user --socket=cups org.libreoffice.LibreOffice` then start application as usual.

**Comment by mapommfj** [comment]
Thanks for this command. Is there a bug or a bad configuration of my printer ?

**Comment by Erick555** [comment]
I think missing this as default in LibreOffice flatpak may be a bug

**Comment by bam80** [comment]
> You can make it permanent with flatpak override --user --socket=cups org.libreoffice.LibreOffice then start application as usual.

Doesn't work for me for some reason. `cupsd` service just doesn't get activated.
Manually run it with `systemctl start` "solves" the problem.
